generate.py

```diff
@@ -91,10 +101,44 @@ def generate_message(media, target, conf, result):
         raise RuntimeError('Invalid media {} for {}'.format(media, target))
 
 
+def count_tweets_in_twitter_url(url):
+    useragent = ["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36","Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36","Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36","Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36","Mozilla/5.0 (Windows NT 10.0; WOW64; rv:49.0) Gecko/20100101 Firefox/49.0","Mozilla/5.0 (Windows NT 6.1; WOW64; rv:49.0) Gecko/20100101 Firefox/49.0","Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12) AppleWebKit/602.1.50 (KHTML, like Gecko) Version/10.0 Safari/602.1.50"]
+    headers = {'User-Agent': random.choice(useragent)}
+    r = requests.get(url, headers=headers)
+    print(url,r.text.count('js-tweet-text-container'))
+    return r.text.count('js-tweet-text-container')
+
+def get_tweets(handle):
+    # get_tweets returns the number of tweets that are have tweeted to the handle about ipv6
+    # for the last NUMBER_OF_YEARS_TO_GET_TWEETS years.
+    # i.e. get_tweets("facebook") returns 133 (01/01/2013 to 10/26/2016)
+    now = datetime.datetime.now()
+    dates_to_try = []
+    for year in range(now.year - NUMBER_OF_YEARS_TO_GET_TWEETS, now.year):
```

Comment:
I'm not really in favor of this apporach.. You can just use the built-in datetime objects for this:

This code does pretty much the same.
```python
import datetime

YEARS = 3
ranges = []

now = datetime.datetime.now()
past = now - datetime.timeldelta(days=365 * YEARS)
delta = datetime.timeldeta(weeks=4)
c = past
while c < now:
   next = c + delta
   ranges.push((c, next))
   c = next

for start, end in ranges:
   print(start, end)
```

This might not be required at all if we are using the API. If it support pagination we might even know the amount of tweets without iterating through them.